spec/factories/items.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :item do
-    name { Faker::Name.name }
+    name { Faker::Commerce.product_name }
```

Comment:
I also find myself playing around with Faker themes to make it fit as close to what the actual attribute is.